components/script/dom/gpuadapter.rs

```diff
@@ -122,15 +151,38 @@ impl AsyncWGPUListener for GPUAdapter {
             Ok(WebGPUResponse::RequestDevice {
                 device_id,
                 queue_id,
-                _descriptor,
+                descriptor,
                 label,
             }) => {
+                let limits = GPULimits {
```

Comment:
We could do that, but `GPULimits` doesn't implement `Clone` so I'd still have to create the new `GPULimits` object (for preserving) manually. Additionally, we would have to add another field to `GPUAdapter` which would store the limits for the device so that when the response arrives we can use them.

Reply:
ok, nevermind then :)